react/src/components/home/Home.jsx

```diff
@@ -67,22 +68,25 @@ function Home() {
       }
     });
 
-    foodResourcesService
-      .getFoodResources()
-      .then(onGetFoodResourcesSuccess)
-      .catch(onGetFoodResourcesError);
-  }, []);
+    const fetchFoodResources = async () => {
+      try {
+        const data = await getFoodResources();
+        await setResultsArray(() => [data]);
+        await setResults(() => [data]);
+      } catch (error) {
+        console.error("Error loading food resources.", error);
+        await setResultsArray(() => [...sampleResults]);
+        await setResults(() => [...sampleResults]);
+      }
+    };
 
-  const onGetFoodResourcesSuccess = (response) => {
-    setResultsArray(() => [...response.data]);
-    setResults(() => [...response.data]);
-  };
+    const resultSetter = async () => {
+      await setResultsArray(() => [...sampleResults]);
+      await setResults(() => [...sampleResults]);
+    };
 
-  const onGetFoodResourcesError = (error) => {
-    console.error("Error!!!!!!!!!!!!!!! ", error);
-    setResultsArray(() => [...sampleResults]);
-    setResults(() => [...sampleResults]);
-  };
```

Comment:
Isn't this the same thing? Axios calls are asynchronous by default so its like your calling an asynchronous method with and asynchronous method with a try and catch. The .then().catch() is a promise that does the same thing as the try can catch? It looks like you moved all the logic into the try can and catch instead of leaving it separated.